src/main/java/service/FileWriterCsvImpl.java

```diff
@@ -0,0 +1,28 @@
+package service;
+
+import static db.Storage.fruits;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class FileWriterCsvImpl implements FileWriterService {
+    private static final String HEADER_FRUIT = "fruitTransaction,";
+    private static final String HEADER_QUANTITY = "quantity";
+
+    @Override
+    public void fileWriter(String filePath) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(filePath))) {
+            bufferedWriter.write(HEADER_FRUIT);
+            bufferedWriter.write(HEADER_QUANTITY);
+            bufferedWriter.newLine();
+            for (Map.Entry<String, Integer> map : fruits.entrySet()) {
+                bufferedWriter.write(map.getKey() + " " + map.getValue());
+                bufferedWriter.newLine();
+            }
+        } catch (IOException exception) {
+            throw new RuntimeException("System can't write data to the file: " + filePath);
+        }
+    }
```

Comment:
![image](https://user-images.githubusercontent.com/20326858/229345627-dd6fbef1-5770-40a7-8533-65ff274711fd.png)
Creating report and writing it to the file are two completely different operations. Do not have them in one class.